packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -5,6 +5,8 @@ import { Platform } from "./platform";
 
 export type ChildProcess = ExecaChildProcess<string>;
 
+const PRE_PATH_TOKEN = "{RNIDE_PRE_PATH}";
```

Comment:
it is no longer "PRE". Can we just name it RNIDE_PATH_DELIMITER. Perhaps we can also just define it in the method as it isn't really used outside of it